src/pages/[productSlug]/tutorials/[collectionSlug]/[...tutorialSlug]/index.tsx

```diff
@@ -33,38 +33,38 @@ async function getStaticPaths(): Promise<
 		}
 	}
 
-	let paths = []
+	// let paths: $TSFixMe[] = []
```

Comment:
TODO: The returned params interface of `getTutorialPagePaths` above is no longer compatible with this route change.

Reply:
Potential change:

```diff
export interface TutorialPagePaths {
	params: {
		productSlug: string
+		collectionSlug: string
		tutorialSlug: string[]
	}
}
```